app/src/serializers/area.serializer.js

```diff
@@ -26,10 +26,32 @@ const areaSerializer = new JSONAPISerializer('area', {
 
 class AreaSerializer {
 
-    static serialize(data) {
-        return areaSerializer.serialize(data);
+    static serialize(data, link = null) {
+        let result;
+        if (data.docs) {
+            result = areaSerializer.serialize(data.docs);
+        } else {
+            result = areaSerializer.serialize(data);
+
+        }
```

Comment:
I think you shouldn't need this `if` statement in this case (in v2 serializer it is needed, however) - `result = areaSerializer.serialize(data);` should be enough